audio-worklet/lib/param-controller.js

```diff
@@ -35,7 +35,17 @@ class ParamController {
     this.id_ = options.id || this.name_;
     this.controller_.type = options.type || 'range';
     this.controller_.min = options.min || 0;
-    this.controller_.max = options.max || 0;
+
+    if (options.max == null)
+      this.controller_.max = 1;
+    else
+      this.controller_.max = options.max;
```

Comment:
Well this is the case with every parameter which has a default value assigned when it is unspecified. The only difference here is that I make the comparison to null explicit. Perhaps the right move is to document the "@param {Object} options", saying that each of its members can be null which will result in default values taking effect?